src/edfi-paging-test/edfi_paging_test/performance_tester.py

```diff
@@ -30,6 +33,17 @@ def _generate_output_reports(args: MainArguments) -> None:
         OutputFormat.JSON: reporter.create_statistics_json
     }
     create_statistics_out[args.contentType](df, args.output, run_name)
+    summary_df = DataFrame(
+        [
+            Summary(
+                key=run_name,
+                machine_name=socket.gethostname(),
+                resources=args.resourceList,
+                description=args.description
+            )
+        ]
+    )
```

Comment:
There's no need to change at this point... but here's a another way that this could have been done:
```suggestion
    summary_df = DataFrame(
        [
           {
                "key": run_name,
                "machine_name": socket.gethostname(),
                "resources": args.resourceList,
                "description": args.description
            }
        ]
    )
```